tutorials/tour/basics.md

```diff
@@ -90,33 +91,33 @@ println({
 
 ## Functions
 
-Functions are expressions with parameters.
+Functions are expressions that take parameters and compute a result.
```

Comment:
Oh, I think rephrasing it to "Functions are expressions that take parameters" is an improvement.

By saying "Functions are expressions with parameters", I tried to communicate that functions are, in essence, just expressions with parameters added in its scope. But I think "that take parameters" communicates it clearer.

I just thought, maybe, "and compute a result" is redundant because expressions always compute results.